lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -0,0 +1,75 @@
+module Fastlane
+  module Actions
+    class AndroidDownloadFileByVersionAction < Action
+      def self.run(params)
+        require_relative '../../helper/android/android_localize_helper.rb'
+        require_relative '../../helper/github_helper'
+
+        version = Fastlane::Helper::Android::LocalizeHelper.get_library_version_from_gradle_config(import_key: params[:import_key])
+        UI.user_error!("Can't find any reference for key #{params[:import_key]}") if version.nil?
+        UI.message "Downloading #{params[:file_path]} from #{params[:repository]} at version #{version} to #{params[:download_folder]}"
+
+        Fastlane::Helper::GithubHelper.download_file_from_release(
+          repository: params[:repository],
+          release: "v#{version}",
```

Comment:
I don't think any of our repos use the `v` prefix for release names… or do they? 🤔 

```suggestion
          release: version,
```

Reply:
Oh, right! `mobile-gutenberg`, which is the repo which we need this right now for, actually uses the `v` prefix. 
I've added better handling for both cases.